metaseq/criterions/cross_entropy.py

```diff
@@ -172,9 +172,15 @@ def reduce_metrics(logging_outputs, data_pruning_metrics=None, data_pruning_metr
                 "ppl", lambda meters: utils.get_perplexity(meters["loss"].avg)
             )
 
-        def serialize_tensor(t, multiplier=1):
-            return [int(x * multiplier) for x in t.cpu().detach().numpy().tolist()]
+        # rounding to nearest int
+        def serialize_tensor(t):
+            return [int(x) for x in t.cpu().detach().numpy().tolist()]
 
+        # rounding to int + 5 decimal digits
```

Comment:
With this we'll have versions of ppl data with different multiplliers. Could we keep it at 10?

Reply:
I think we might have been using 10000 for a while? I remember we had two versions of this function in the same file, and I put the `* 10000` version for our files a while back because I wanted to see a bit more decimal points. We can change back if you want to